docs/internals/frontends.rst

```diff
@@ -152,8 +245,222 @@ username
 comment
     Archive comment, if any
 
+Example of a simple archive listing (``borg list --last 1 --json``)::
+
+    {
+        "archives": [
+            {
+                "archive": "2017-02-27T21:21:51",
+                "barchive": "2017-02-27T21:21:51",
```

Comment:
hmm? this is either the same as name or (if it would be non-utf8 binary somehow) you could not include it anyway?